PraceDomowe/PracaDomowa6/Swiatek_Patryk/Praca_domowa_6.ipynb

```diff
@@ -0,0 +1,166 @@
+{
```

Comment:
<p>Bardzo dobrze, chociaż skoro to jest szereg czasowy to ja bym rozważył użycie wykresu liniowego.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-ExploratoryDataAnalysis/pull/191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>